### ISSUE TITLE
showing scale / axes visual in empty viewer on startup can segfault

### ISSUE THREAD

**Issue by sofroniewn** [napari/napari#1881]
## 🐛 Bug

It was noted in #1835 that showing a scale or axes visual (using the menubar) after launching napari from the command line with `napari` can trigger a segfault. Commenting out the text node in the welcome visual prevents it from happening.

```python-traceback
(base) czirwc1macos2701:napari nsofroniew$ napari
WARNING: Error drawing visual <vispy.visuals.line.line._GLLineVisual object at 0x7faa2ba51588>
18:17:02 WARNING Error drawing visual <vispy.visuals.line.line._GLLineVisual object at 0x7faa2ba51588>
/Users/nsofroniew/opt/anaconda3/bin/pythonw: line 3: 63069 Segmentation fault: 11  /Users/nsofroniew/opt/anaconda3/python.app/Contents/MacOS/python "$@"
```

Once some visuals have been added to the viewer this doesn't seem to happen, even if the viewer is made empty again

**Comment by andy-sweet** [comment]
@sofroniewn : similar to https://github.com/napari/napari/issues/1905, do you think this was also fixed by #2542?

**Comment by sofroniewn** [comment]
yup - fixed, just tried it and no error - will close. thanks!!
